client/src/components/MoneyButton/MoneyButton.style.js

```diff
@@ -21,8 +26,17 @@ export const Order = styled.section`
     padding: 0;
     margin: 0;
   }
-  display: flex;
-  justify-content: space-between;
+`;
+
+export const Checkout = styled.button`
```

Comment:
This could be extended from AddToBasket styles as they share a lot of properties.

Reply:
Solved